contracts/contracts/vault/VaultCore.sol

```diff
@@ -340,11 +317,7 @@ contract VaultCore is VaultStorage {
             // No balance, nothing to do here
             if (assetBalance == 0) continue;
 
-            // Multiply the balance by the vault buffer modifier and truncate
-            // to the scale of the asset decimals
-            uint256 allocateAmount = assetBalance.mulTruncate(
-                vaultBufferModifier
-            );
+            uint256 allocateAmount = assetBalance;
```

Comment:
nit: is this intermediary variable necessary? How about renaming assetBalance on line 316 to allocateAmount?

Reply:
Cleaner. Done.